include/common/s1ap_structs.h

```diff
@@ -610,6 +610,10 @@ typedef enum s1apCauseMisc {
     s1apCauseMisc_unknown_PLMN  = 5
 } e_s1apCauseMisc;
 
+typedef enum emmCause {
```

Comment:
These header files really need cleanup. Current code has enums/defines everywhere...We have to align it nicely in header files which have meaningful names.  Can you open git issue or jira issue. Someday we have to cleanup the code.

Reply:
Created cleanup task EDGEPOD-279.